topics/admin/tutorials/job-destinations/tutorial.md

```diff
@@ -137,45 +143,133 @@ We don't want to overload our training VMs trying to run real tools, so to demon
+> 1. Edit `group_vars/galaxyservers.yml` and add the following destination under your `job_config` section to route all jobs to TPV.
 >
 >    {% raw %}
 >    ```diff
 >    --- a/group_vars/galaxyservers.yml
 >    +++ b/group_vars/galaxyservers.yml
->    @@ -52,9 +52,22 @@ galaxy_job_config:
->               value: /tmp/singularity
->             - name: SINGULARITY_TMPDIR
->               value: /tmp
->    +      slurm-2c:
->    +        runner: slurm
->    +        singularity_enabled: true
->    +        native_specification: --nodes=1 --ntasks=1 --cpus-per-task=2
->    +        env:
->    +        - name: LC_ALL
->    +          value: C
->    +        - name: SINGULARITY_CACHEDIR
->    +          value: /tmp/singularity
->    +        - name: SINGULARITY_TMPDIR
->    +          value: /tmp
->       tools:
->         - class: local # these special tools that aren't parameterized for remote execution - expression tools, upload, etc
->           environment: local_env
->    +    - id: testing
->    +      environment: slurm-2c
->     
->     galaxy_config:
->       galaxy:
+>    @@ -32,34 +32,15 @@ galaxy_config:
+>           handling:
+>             assign: ['db-skip-locked']
+>           execution:
+>    -        default: singularity
+>    +        default: tpv_dispatcher
+>             environments:
+>    -          local_env:
+>    -            runner: local_runner
+>    -            tmp_dir: true
+>    -          singularity:
+>    -            runner: local_runner
+>    -            singularity_enabled: true
+>    -            env:
+>    -            # Ensuring a consistent collation environment is good for reproducibility.
+>    -            - name: LC_ALL
+>    -              value: C
+>    -            # The cache directory holds the docker containers that get converted
+>    -            - name: SINGULARITY_CACHEDIR
+>    -              value: /tmp/singularity
+>    -            # Singularity uses a temporary directory to build the squashfs filesystem
+>    -            - name: SINGULARITY_TMPDIR
+>    -              value: /tmp
+>    -          slurm:
+>    -            runner: slurm
+>    -            singularity_enabled: true
+>    -            env:
+>    -            - name: LC_ALL
+>    -              value: C
+>    -            - name: SINGULARITY_CACHEDIR
+>    -              value: /tmp/singularity
+>    -            - name: SINGULARITY_TMPDIR
+>    -              value: /tmp
+>    +          tpv_dispatcher:
+>    +            runner: dynamic
+>    +            type: python
+>    +            function: map_tool_to_destination
+>    +            rules_module: tpv.rules
+>    +            tpv_config_files:
+>    +              - config/tpv_rules_local.yml
```

Comment:
tpv could only find this when I listed it with the absolute path:
`- "{{ galaxy_config_dir }}/tpv_rules_local.yml"